internal/utils/fs.go

```diff
@@ -0,0 +1,59 @@
+package utils
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+// CopyFile copies the file from the source to the destination
+func CopyFile(src, dest string) error {
+	// open source file
+	srcFile, err := os.Open(src)
+	if err != nil {
+		return fmt.Errorf("error opening source file: %s", err)
+	}
+
+	// open destination file
+	destFile, err := os.Create(dest)
+	if err != nil {
+		srcFile.Close()
+		return fmt.Errorf("error opening destination file: %s", err)
+	}
+	defer destFile.Close()
```

Comment:
Here we probably shouldn't use `defer`, see https://www.joeshaw.org/dont-defer-close-on-writable-files/